### DISCUSSION TITLE
Volume Control

### DISCUSSION THREAD

**Discussion by poutineflyer** [TannerGabriel/discord-bot#84] [Ideas]
is there any possibly you could add a volume control command?Thanks

**Comment by TannerGabriel** [comment]
@poutineflyer Yes, that is certainly possible. I'm currently short on time but can add it when I find some time (Maybe this weekend).

**Reply by Plaz5** [reply]
To add to this, would you be able to add a feature to play, so that when the bot joins the voice channel its volume can be preset to a certain level. Personally I would default this to .5 (50%) as bots that come in at full volume are super loud.
